Don't assume tinyxml in /usr. The patch is a bit rough: robotpkg passes the
necessary variables.

--- CMakeLists.txt.orig	2013-06-06 20:59:44.000000000 +0200
+++ CMakeLists.txt	2013-06-28 00:29:43.000000000 +0200
@@ -5,18 +5,15 @@
 find_package(Boost REQUIRED COMPONENTS filesystem program_options system)
 set(Python_ADDITIONAL_VERSIONS 2.7)
 find_package(PythonLibs REQUIRED)
-find_package(tinyxml QUIET)
-if(NOT tinyxml_FOUND)
-  # Just assume it's located in the system directories
-  # though you could try and detect stuff via the pkgconfig file
-  # it provides
-  set(tinyxml_LIBRARIES tinyxml)
-endif() 
+find_file(tinyxml_HEADER tinyxml.h PATH ${TINYXML_ROOT}/include ${TINYXML_ROOT}/include/tinyxml)
+find_library(tinyxml_LIBRARIES libtinyxml.so PATH ${TINYXML_ROOT}/lib)
+get_filename_component(tinyxml_INCLUDE_DIRS ${tinyxml_HEADER} PATH)
+set(tinyxml_FOUND TRUE)
 
 catkin_package(
   INCLUDE_DIRS include
-  LIBRARIES rospack tinyxml ${PYTHON_LIBRARIES}
-  DEPENDS Boost
+  LIBRARIES rospack ${PYTHON_LIBRARIES}
+  DEPENDS Boost tinyxml
 )
 
 #add_definitions(-Wall)
